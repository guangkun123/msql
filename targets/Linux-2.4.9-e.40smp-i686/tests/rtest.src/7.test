../../../..//src/tests/rtest.src/7.test