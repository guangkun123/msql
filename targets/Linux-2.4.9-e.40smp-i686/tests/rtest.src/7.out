../../../..//src/tests/rtest.src/7.out